File: Imp-Java-c/c/Sort/bubblesort.c
#include <stdio.h>

void bubblesort()
{
    int array[] = {4, 1, 6, 2, 8, 7};

    for(int i=0; i<6; i++){
        for(int j = 0; j < 6; j++){
            if(array[j] > array[j+1]){
                int temp = array[j];
                array[j] = array[j+1];
                array[j+1] = temp;
            }
        }
    }
    for(int i=0; i<6; i++)
    {
        printf("%d ", array[i]);
    }
}

File: Imp-Java-c/java/BubbleSort.java
import java.util.*;

public class BubbleSort {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] A = new int[n];
        for (int i = 0; i < n; i++)
            A[i] = sc.nextInt();
        bubbleSort(A);
        System.out.println(Arrays.toString(A));
    }

    static void bubbleSort(int[] A) {
        for (int i = 1; i <=A.length-1; i++) {
            for (int j = 0; j <=(A.length-i-1); j++) {
                if (A[j] > A[j + 1]) {
                    int temp = A[j];
                    A[j] = A[j + 1];
                    A[j + 1] = temp;
                }
            }
        }
    }
}